hub/jupyterhub_config.py

```diff
@@ -13,6 +13,28 @@
 # Our simplest user image! Optimized to just... start, and be small!
 c.KubeSpawner.singleuser_image_spec = 'yuvipanda/simple-singleuser:v1'
 
+# Configure dynamically provisioning pvc
+c.KubeSpawner.pvc_name_template = 'claim-{username}-{userid}'
+c.KubeSpawner.storage_class = 'single-user-storage'
```

Comment:
Specify the specification we want the pvc to use (i.e. which zone and type of claim on GoogleCloud). This is defined in the ```StorageClass``` of the manifestfile